library/src/main/java/bio/terra/landingzone/stairway/flight/create/resource/step/CreateAksStep.java

```diff
@@ -132,8 +140,43 @@ private KubernetesCluster createAks(FlightContext context) {
                       LandingZoneTagKeys.LANDING_ZONE_ID.toString(),
                       landingZoneId.toString(),
                       LandingZoneTagKeys.LANDING_ZONE_PURPOSE.toString(),
-                      ResourcePurpose.SHARED_RESOURCE.toString()))
+                      ResourcePurpose.SHARED_RESOURCE.toString(),
```

Comment:
Seems this tag should be conditionally set based on costSavingsEnabled parameter